packages/fxa-payments-server/src/components/PlanDetails/index.scss

```diff
@@ -1,14 +1,23 @@
 @import '../../../../fxa-content-server/app/styles/variables';
+@import '../../../../fxa-content-server/app/styles/breakpoints';
 
 .plan-details-component {
-  background: $color-white;
-  border-radius: 8px;
-  box-shadow: $card-box-high;
   font-size: 15px;
-  margin: 0 auto;
-  padding: 20px 20px 0;
+  margin: 65px auto 0;
   width: 342px;
 
+  @include min-width('tablet') {
```

Comment:
This is more of a BTW comment I could have added in another review: mixing the old breakpoints (`respond-to`) with the newer breakpoints from `min-width` might produce strange results, ideally we use one set or the other set across the board IMO, but if things are looking good at major breakpoints then this seems OK 👍